assets/src/scss/pages/post/_post.scss

```diff
@@ -2,19 +2,16 @@
 .single-attachment,
 .single-idea {
   @include large-and-up {
-    .post-content,
-    .author-block,
-    .page-header {
+    .narrow-container {
```

Comment:
Now that the class is only present where we include it, we can move it outside of the parent selector.